tests/spec/marathon/marathon_spec.rb

```diff
@@ -24,6 +24,9 @@
 
 describe command("curl -s -XPOST 127.0.0.1:8080/v2/apps -d@spec/marathon/sample.json -H \"Content-Type: application/json\" && sleep 10") do
   its(:exit_status) { should eq 0 }
+  it {
```

Comment:
didnt see this. my way works but looks hacky. ill just increase the sleep above this.